newdle/newdle.cfg.example

```diff
@@ -97,3 +97,6 @@ LAST_ACTIVITY_CLEANUP_DELAY = None
 FINAL_DATE_CLEANUP_DELAY = None
 # Days after which a newdle that was deleted by its creator is cleaned up.
 DELETED_CLEANUP_DELAY = None
+
+# The INDICO_URL is used for creating an event from a Newdle. 
```

Comment:
# If set, newdle will show a button to create an Indico event using data from the newdle.

Reply:
We should hide that button btw if no URL is set.

Reply:
I'm not sure how to hide the button. Is there a way to import the config in `.js` files?

Reply:
hm i thought we already had an endpoint to return settings but we don't..

so you need to add something like this:

```
@api.route('/config/')
@allow_anonymous
def config():
    return jsonify(has_event_creation=bool(current_app.config.CREATE_EVENT_URL))
```

and then call it to determine whether to show the button or not

Reply:
Added, thanks.